libraries/helpers.rb

```diff
@@ -28,3 +46,19 @@ def get_delivery_versions(node)
     end
   end
 end
+
+module DeliverySugarExtras
+  module DSL
+    def version_env_key(version)
+      DeliverySugarExtras::Helpers.version_env_key(version)
+    end
+
+    def add_all_change_data_to_node(node)
+      DeliverySugarExtras::Helpers.add_all_change_data_to_node(node)
+    end
+
+    def get_delivery_versions(node)
```

Comment:
I think this method shouldn't pass any attributes so it can be a DSL Method... 

This is currently braking `delivery-cluster` pipeline: 

```
  NoMethodError
  -------------
  undefined method `get_delivery_versions' for DeliverySugarExtras::Helpers:Module
  
  Cookbook Trace:
  ---------------
    /var/opt/delivery/workspace/delivery.chef.co/chef/delivery/delivery-cluster/upgrade_aws/acceptance/provision/cache/cookbooks/build/recipes/provision_upgrade_aws.rb:27:in `from_file'
    /var/opt/delivery/workspace/delivery.chef.co/chef/delivery/delivery-cluster/upgrade_aws/acceptance/provision/cache/cookbooks/delivery-red-pill/recipes/_include_build_cb_recipe.rb:24:in `from_file'
    /var/opt/delivery/workspace/delivery.chef.co/chef/delivery/delivery-cluster/upgrade_aws/acceptance/provision/cache/cookbooks/delivery-red-pill/recipes/provision.rb:49:in `from_file'
    /var/opt/delivery/workspace/delivery.chef.co/chef/delivery/delivery-cluster/upgrade_aws/acceptance/provision/cache/cookbooks/build/recipes/provision.rb:24:in `from_file'
  
  Relevant File Content:
  ----------------------
  /var/opt/delivery/workspace/delivery.chef.co/chef/delivery/delivery-cluster/upgrade_aws/acceptance/provision/cache/cookbooks/build/recipes/provision_upgrade_aws.rb:
  
   20:  
   21:  chef_gem "chef-rewind"
   22:  require 'chef/rewind'
   23:  
   24:  cluster_name     = "#{node['delivery']['change']['stage']}_#{node['delivery']['change']['pipeline']}"
   25:  path             = node['delivery']['workspace']['repo']
   26:  cache            = node['delivery']['workspace']['cache']
   27>> delivery_version = ::DeliverySugarExtras::Helpers.get_delivery_versions(node)[1]
   28:  
   29:  include_recipe "build::provision_clean_aws"
   30:  
   31:  rewind "template[Create Environment Template]" do
   32:    variables(
   33:      :delivery_license => "#{cache}/delivery.license",
   34:      :delivery_version => delivery_version,
   35:      :cluster_name => cluster_name
   36:    )
```
https://delivery.chef.co/e/chef/#/organizations/delivery/projects/delivery-cluster/changes/14a008ac-5844-475d-9e18-55bb90ad6388/status/acceptance
https://delivery.chef.co/e/chef/#/organizations/delivery/projects/delivery-cluster/changes/04405b54-4bf5-4eb1-9709-ce4475719aa0/status/acceptance